### ISSUE TITLE
easycwmp dump

### ISSUE THREAD

**Issue by successbyfailure** [zry98/movistar-home-hacks#1]
Muy buen trabajo so far! Tengo un home nuevo y dos de segunda mano de camino :)
¿Quieres que pruebe algo con easycwmp ahora que están nuevos?

**Comment by zry98** [comment]
Sí, por favor! no sé si tiene un Movistar WiFi para conectarlo, si es así, intenta iniciar sesión en la página de easycwmp y ve si hay algo a explotar. Y mejor escanea los puertos abiertos otra vez, ya que escaneé el mío mientras no estaba en la pantalla principal, creo que hubo muchos apps de Android y servicios que no se iniciaron en ese momento, tal vez hay algo más para explotar.

He subido algunos ficheros relacionados con el easycwmp que encontré en el flash dump, que contienen algunas contraseñas y otras configuraciones. Have fun!

**Comment by successbyfailure** [comment]
De momento he llegado hasta la pantalla de selección de wifi (tengo mvstar pero cambie el router): Se conecta a la red y no pasa de ahí al no reconocer el router de movistar. 
Le he tirado un nmap y solo ha detectado el 7006 abierto que contesta con una firma de easycwmp. por ese puerto he intentado abrir el navegador con easycwmp/easycwmp y no entra,vuelve a pedir el user.

Voy a sacar el router original del armario a ver si pasa de la ventana de seleccion de wifi, aunque me da la sensacion de que el cwmp ya deberia estar activo en esta fase.

**Comment by zry98** [comment]
Hmmm, supongo que aún necesitarás desmontarlo y soldar un cable USB para continuar. Pero podrías probar las credenciales `ACS1234/ACS1234` y `ACSCR1234/ACSCR1234` en el otro fichero? Pero creo que estas son para el servidor ACS.
No puedo encontrar nada más en mi dump, tal vez haya otras configuraciones en la partición cifrada /data.

**Comment by successbyfailure** [comment]
Tambien las he probado y nada. Pero pensándolo no se si es cosa de la clave o que no estoy llamando a ningún endpoint conocido.  ¿tu llegaste a ver algo en esa web en tu home?

**Comment by zry98** [comment]
no lo probé antes de instalar el linux, ahora no puede arrancar el Android-x86 original (pensé que sería fácil hacerlo reescribiendo el dump en flash jaja), ahora se atasca en la pantalla de arranque con fondo azul y un logo blanco de movistar.

**Comment by ppnacho** [comment]
Hola,

Buen proyecto. Acabo de hacerme con uno de estos equipos y estoy valorando reconvertirlo. No soy muy experto, pero si puedo contribuir con cualquier prueba, pues dentro del escaso tiempo libre que tengo, pues me apunto. Un saludo

**Comment by PaskoMoto** [comment]
Hola.
Acabo de recibir dos modelos Mitrastar, con la intención de soldarles el microusb a la placa base. 
Como aún estoy pendiente de recibir los conectores, he estado leyendo la documentación de aquí y he encontrado además ésto:
[https://es.easeus.com/computer-instruction/aptio-setup-utility.html](url)
Se trata de un software que asegura que puede recuperar información en caso de fallos en el arranque y mi pregunta es:
Creéis que puede ser útil para:
a.- ¿Volver a insertar el dump de disco (o una copia que yo pueda extraer) de nuevo en el dispositivo?
b.- ¿Extraer una copia del Android antes de empezar a tocar nada?
c.- ¿Lo conocéis/lo habéis utilizado antes?

Os aviso cuando me disponga a ponerme manos a la obra.
Gracias por vuestro trabajo.

**Comment by zry98** [comment]
Hola @PaskoMoto, gracias por tu interés.

> Acabo de recibir dos modelos Mitrastar, con la intención de soldarles el microusb a la placa base.

Ante todo, estás seguro de que son del [modelo Intel](https://github.com/zry98/movistar-home-hacks#nota-importante)?

> a.- ¿Volver a insertar el dump de disco (o una copia que yo pueda extraer) de nuevo en el dispositivo?
> b.- ¿Extraer una copia del Android antes de empezar a tocar nada?

En teoría sí, pero dudo que sea diferente al comando `dd`.

> c.- ¿Lo conocéis/lo habéis utilizado antes?

En mi caso, lo he visto antes pero todavía no lo he usado.

¡Buena suerte y feliz hackeo!

**Comment by PaskoMoto** [comment]
> Hola [@PaskoMoto](https://github.com/PaskoMoto), gracias por tu interés.
> 
> > Acabo de recibir dos modelos Mitrastar, con la intención de soldarles el microusb a la placa base.
> 
> Ante todo, estás seguro de que son del [modelo Intel](https://github.com/zry98/movistar-home-hacks#nota-importante)?

Si los Mitrastar son sólo Intel, sí, son Intel. Los compré con la intención de ponerles un Linux. ¿Hay modelos Mitrastar con SnapDragon/ARM?
> 
> > a.- ¿Volver a insertar el dump de disco (o una copia que yo pueda extraer) de nuevo en el dispositivo?
> > b.- ¿Extraer una copia del Android antes de empezar a tocar nada?
> 
> En teoría sí, pero dudo que sea diferente al comando `dd`.

:D 
> 
> > c.- ¿Lo conocéis/lo habéis utilizado antes?
> 
> En mi caso, lo he visto antes pero todavía no lo he usado.
> 
> ¡Buena suerte y feliz hackeo!

Gracias. Tengo una pregunta más:
¿Qué ocurre si NO se puentea a GND el pin ID del puerto USB? ¿Deja de funcionar el puerto?

Saludos y gracias.

**Comment by zry98** [comment]
@PaskoMoto 

> ¿Hay modelos Mitrastar con SnapDragon/ARM?

No, lo leí mal.

> ¿Qué ocurre si NO se puentea a GND el pin ID del puerto USB? ¿Deja de funcionar el puerto?

Con el puente, el dispositivo funciona como un host USB, es como un puerto USB normal de tu PC, se puede conectar teclado, disco, etc.
Sin el puente, el dispositivo funciona como un móvil Android cuando se conecta a un PC, en teoría se pueden hacer cosas con ADB pero no lo he investigado porque la instalación de Linux eliminó el bootloader original (para el Android-x86).
